src/stores/filingHistoryListStore.ts

```diff
@@ -29,9 +29,11 @@ export const useFilingHistoryListStore = defineStore('filingHistoryList', {
       // return only the valid filings
       return state.filings.filter(filing => {
         // safety check for required fields
-        if (!filing.name || !filing.displayName || !filing.effectiveDate || !filing.submittedDate || !filing.status) {
-          // eslint-disable-next-line no-console
-          console.log('Invalid filing =', filing)
+        if (!filing.name || !filing.displayName || !filing.effectiveDate || !filing.submittedDate || !filing.status || !filing.displayLedger) {
+          if (filing.displayLedger) {
+            // eslint-disable-next-line no-console
+            console.log('Invalid filing =', filing)
+          }
```

Comment:
No, don't do it this way. This is mixing the logic in line 32 with the logic in line 33. I think it would be cleaner to have two separate if statements, please.

Reply:
Looks good now. Thanks.